cmd/scollector/collectors/iterable.go

```diff
@@ -0,0 +1,70 @@
+package collectors
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"github.com/bosun-monitor/statusio"
+)
+
+func init() {
+	registerInit(func(c *conf.Conf) {
+		collectors = append(collectors, &IntervalCollector{
+			F: func() (opentsdb.MultiDataPoint, error) {
+				return cIterableStat(c.Iterable.StatusBaseAddr)
+			},
+			name:     "cIterableStat",
```

Comment:
Yes, I hesitated there. I suggest one day we rename all of them as it makes golint output unreadable.
Collectors should be bounded to this project anyway, so that would be doable.